benchmark/run.rb

```diff
@@ -19,6 +19,9 @@ module GraphQLBenchmark
   BIG_SCHEMA = GraphQL::Schema.from_definition(File.join(BENCHMARK_PATH, "big_schema.graphql"))
   BIG_QUERY = GraphQL.parse(File.read(File.join(BENCHMARK_PATH, "big_query.graphql")))
 
+  FIELDS_WILL_MERGE_SCHEMA = GraphQL::Schema.from_definition(File.join(BENCHMARK_PATH, "fields_will_merge_schema.graphql"))
```

Comment:
Since the schema here is so simple, how about inlining the string instead? I imagine my future self will appreciate seeing the relevant text without opening another file. 

```suggestion
  FIELDS_WILL_MERGE_SCHEMA = GraphQL::Schema.from_definition("type Query { hello: String }")
```